src/main/resources/ApplicationResources_zh.properties

```diff
@@ -1428,13 +1428,58 @@ restore.resource.success=由於發布失敗，還原資料集 {1} 至版本 \#{0
 restore.resource.failed=還原資料集 {1} 版本 \#{0} 失敗：{2}
 
 # Auto-publishing
-autopublish=自動定期發布
```

Comment:
We need to explore the implications of CrowdIn here.
I am not sure if CrowdIn pulls changes like this in, or if we need to start there and let it merge the changes from the English one

Reply:
Oh I just look at the history of this file, and I notice there is a Crowdin pipeline that automatically commit this file.

I put the key and english values in all non-english files, because otherwise, the key are displayed when we select another language than english (English is not the base language, overriden by the other language files, as it is usually !)